mssqlcli/main.py

```diff
@@ -52,19 +45,26 @@ def run_cli_with(options):
     mssqlcli.connect_to_database()
 
     telemetry_session.set_server_information(mssqlcli.mssqlcliclient_main)
-    mssqlcli.run()
+
+    # don't run cli if query is specified
+    if not options.query:
+        mssqlcli.run()
+    else:
+        mssqlcli.execute_query(str(options.query))
+        mssqlcli.shutdown()

```

Comment:
probably should wrap this in a try finally block so that we always call shutdown